src/components/edit-movie/edit-movie.tsx

```diff
@@ -2,30 +2,81 @@ import classNames from 'classnames';
 import { AppButton } from 'components/app-button/app-button';
 import { Genres } from 'components/menu/menu';
 import { Modal } from 'components/modal/modal';
+import { useState } from 'react';
+import { useAppDispatch } from 'redux/hooks';
+import { addMovieAction, updateMovieAction } from 'redux/movies';
+import { Movie } from 'types/movie';
 import styles from './edit-movie.module.scss';
 
 const genres = [Genres.Comedy, Genres.Crime, Genres.Documentary, Genres.Horror];
 
+type MovieFormValues = Partial<Movie>;
```

Comment:
It can be written as:
```suggestion
interface MovieFormValues extends Partial<Movie> { }
```

Reply:
![image](https://user-images.githubusercontent.com/69203629/209430150-6ecc13bd-15ac-4045-bf0c-bd1d386ed25c.png)